lib/global-styles.php

```diff
@@ -708,7 +743,7 @@ function gutenberg_experimental_global_styles_settings( $settings ) {
 	// STEP 1: ADD FEATURES
 	// These need to be added to settings always.
 	// We also need to unset the deprecated settings defined by core.
-	$settings['__experimentalFeatures'] = gutenberg_experimental_global_styles_get_editor_features( $merged );
+	$settings['__experimentalFeatures'] = gutenberg_experimental_global_styles_get_editor_settings( $merged );
```

Comment:
Should we continue to call all the settings including the presets as features?

Reply:
Looks like I made the same comment here https://github.com/WordPress/gutenberg/pull/25301#discussion_r488020497